### ISSUE TITLE
How should global parameters be passed

### ISSUE THREAD

**Issue by gsqrt2** [nuxt-community/storybook#110]
I am trying to pass global parameters to stories. The non-nuxtjs way is to add the desired data in .storybook/preview.js, but in this case the respective file ./nuxt-storybook/storybook/preview.js is overwritten in build time. I have also tried to pass them as a storybook option in nuxt.config.js without success.

Unfortunately there is no reference in the documentation. How should we handle this?

Sidenote: Addon options seem not to work as documented as well.

The documentation's example:
`export default {
  storybook: {
    addons: [
      {
        name: '@storybook/preset-scss',
        options: {
          cssLoaderOptions: {
              modules: true,
              localIdentName: '[name]__[local]--[hash:base64:5]',
          }
        }
      }
    ]
  }
}`

It throws the error: Cannot find module '[object Object]' the moment you insert an object instead of a string in the addons[] array.

**Comment by farnabaz** [comment]
Hey @gsqrt2,
This is fixed in https://github.com/nuxt-community/storybook/pull/103 and will publish ASAP

**Comment by gsqrt2** [comment]
That's great @farnabaz!
I see that #103 is about addon options. Would it also include the global parameter functionality?
Thanks!

**Comment by fbtwitter** [comment]
May i open this this issue again, i had an issue each time im using global parameter, all those dynamic stories in docs tab are disappearing.

i'm using storybook eject. every add-on and custom background loaded. except those dynamic stories in docs.

**Comment by fbtwitter** [comment]
I wanted to mention this merged version #169 v.3.2

after having auto upgrade to version 3.3.1

this is what i got whenever declare parameters in preview.js

![image](https://user-images.githubusercontent.com/33839891/105571335-c7d93f80-5d81-11eb-93e8-cdeedb58b21c.png)

and in the docs section, the dynamic stories won't loaded
![image](https://user-images.githubusercontent.com/33839891/105571347-dfb0c380-5d81-11eb-9c3e-f7e514531ec6.png)

i'm a bit not used to.. drawn the reproduction step in codesandbox, this what I have tried,
[https://codesandbox.io/s/suspicious-river-f9vb1?file=/components/AHeading/index.vue](url)

**Comment by farnabaz** [comment]
Hey, Sorry for the late response @fbtwitter 
In your `preview.js` you should merge Nuxt parameters with your custom parameters.

```
import { parameters as nuxtParameters } from "~~/.nuxt-storybook/storybook/preview.js";
export * from "~~/.nuxt-storybook/storybook/preview.js";

export const parameters = {
  nuxtParameters,
  actions: { argTypesRegex: "^on[A-Z].*" },
  backgrounds: {
    default: "talkiebuddy",
    values: [
      {
        name: "talkiebuddy",
        value: "#FFFCF2"
      },
      {
        name: "white",
        value: "#FFFFFF"
      },
      {
        name: "dark",
        value: "#1e2225"
      }
    ]
  }
};

```
